### DISCUSSION TITLE
最上段キーを押すと同じ行が全て反応します

### DISCUSSION THREAD

**Discussion by mcoods** [Yowkees/keyball#713] [Q&A]
右手部分の最上段キーを押すと、同じ縦の行が全て(例えばyを押すとy,h,nが)反応してしまいます

**Comment by mcoods** [comment]
![20241220_005211](https://github.com/user-attachments/assets/4f9b7d7c-19ce-4a07-8915-ecd97fc678a7)
画像を添付し忘れていたのでこちらに載せさせていただきます

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

最上段のどのキーを押しても、縦１列反応しますか？例えば、uを押すとj,mも、iを押すとk,,が入力されてしまうという理解で良かったでしょうか。
その場合、私の思いつくのが、ProMicro裏でのROW0,ROW1,ROW2の短絡です。
ProMicro本体とProMicro裏の半田付けで、真ん中付近（ROW0,ROW1,ROW2のシルクがある周辺）で隣とショートしている端子がないかご確認をお願いします。

以上、お手数ですがご対応お願いいたします。

**Reply by mcoods** [reply]
返信ありがとうございます。
promicroのはんだを一部除去したら解決いたしました。ありがとうございます。
